packages/elements/src/components/progress-bar/progress-bar.component.ts

```diff
@@ -0,0 +1,146 @@
+import { MutationTestMetricsResult } from 'mutation-testing-metrics';
+import { RealTimeElement } from '../real-time-element';
+import { PropertyValues, html, nothing } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import { tailwind } from '../../style';
+
+interface ProgressBarMetrics {
+  killed: number;
+  survived: number;
+  combined: number;
+  [key: string]: number;
+}
```

Comment:
See my previous remark. By removing this interface, the progress bar can be a lot simpler.